packages/core/test/template-engine/built-in-extensions/cache/cache.spec.ts

```diff
@@ -174,3 +174,41 @@ select count(*) as count from vulcan.user where user.id = {{ context.params.user
   expect(binding[0].get('$1')).toBe(`user-id`);
   expect(resultData).toEqual([{ count: 1 }]);
 });
+
+it('Extension should remove comments in sql statements', async () => {
```

Comment:
Please assist to add the multiple line comment test case, thanks !

Reply:
added multi-line comments in the test case